script/Character.py

```diff
@@ -1,36 +1,45 @@
 # Program imports
 import random
 
-from script import Items
+import Items
 from Mechanics.ui_mechanics import *
 from Map_Gen.ParseMap import WorldMap
 
 
 class Character:
-    def __init__(self, name, max_health, melee_attack, magic_attack,
-                 max_mana, max_stamina, defense, luck, level=1):
+    def __init__(self, name=None, max_health=1, health=max_health, melee_attack=None, magic_attack=None,
```

Comment:
Some of these don't seem like sensible defaults for a Character to have. Is there a reason we want to have them? Sometimes it is a good thing to force the user to provide necessary valid attribute values when initialising a character.

Reply:
Until we determine exactly what that looks like, I think its best we at least initialize to None or 1. Right now its about writing functional code. As we start to see all the code come together more and more, we can refactor and create default initial values as needed. For now, I think it should stay this way for testing purposes.